web/themes/custom/votegov/templates/navigation/breadcrumb.html.twig

```diff
@@ -11,10 +11,14 @@
   <nav role="navigation" aria-labelledby="system-breadcrumb" class="usa-breadcrumb">
     <h2 id="system-breadcrumb" class="usa-sr-only visually-hidden">{{ 'Breadcrumb' | t }}</h2>
     <ol class="usa-breadcrumb__list">
-    {% for item in breadcrumb %}
+    {% for key,item in breadcrumb %}
       <li class="usa-breadcrumb__list-item">
         {% if item.url %}
-          <a class="usa-breadcrumb__link" href="{{ item.url }}"><span>{{ item.text }}</span></a>
+          {% if key is same as(0) %}
+            <a class="usa-breadcrumb__link" href="{{ item.url }}"><span>{{ 'Home'|t }}</span></a>
+          {% else %}
+            <a class="usa-breadcrumb__link" href="{{ item.url }}"><span>{{ item.text }}</span></a>
+          {% endif %}
         {% else %}
           {{ item.text }}
```

Comment:
You can remove the `key`. 
The `item.text` is already present we just need to add the `| t` to it.
For both with and without a link add the `t` function like this: `{{ item.text | t }}`.

Reply:
@mlloydbixal this feedback is still pending.